ang/civicase/case/details/people-tab/services/people-tab-roles.service.js

```diff
@@ -180,7 +180,7 @@
             end_date: null,
             email: contact.email,
             phone: contact.phone,
-            role: contact.role,
+            role: ts('Client'),
```

Comment:
Would `ts(contact.role)` work here? Should we use this instead?

Reply:
@ivan-compucorp it would work, but that's not the recommended way to use the `ts` according to the official guide:
https://docs.civicrm.org/dev/en/latest/translation/#best-practices

> You must pass a literal string into ts(), not a variable.